### PR TITLE
Implementing Simple Timer

### PR DESCRIPTION
**Pull request by adityashahms**
**Task name**: AzureTestPlanV0

**Description**: Telemetry for execution time in tasks

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

**Local Testing**:
- Sample Telemetry Body
image
- Task log
image

### CODE REVIEW

File: Tasks/AzureTestPlanV0/constants.ts [hunk 1]
```diff
@@ -9,3 +9,6 @@ export const AUTOMATION_STATUS = 'Microsoft.VSTS.TCM.AutomationStatus';
 export const REVISION_ID = 'System.Rev';
 export const NOT_AUTOMATED = 'Not Automated'
 export const MERGE_THRESHOLD = 100;
+export const AUTOMATED_EXECUTION = "AutomatedExecutionPhase";
+export const AUTOMATED_PUBLISHING = "PublishingAutomatedResultsPhase";
+export const MANUALTESTS_EXECUTION = "ManualTestResultsPublishingPhase";
```
Comment (triptijain2112) [inline Tasks/AzureTestPlanV0/constants.ts:14]: MANUALTESTS_EXECUTION = "ManualTestResultsPublishingPhase"
Execution and publishing are not in line.

File: Tasks/AzureTestPlanV0/SimpleTimer.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { ciDictionary } from "./ciEventLogger";
```
Comment (triptijain2112) [inline Tasks/AzureTestPlanV0/SimpleTimer.ts:1]: Do we need our own timer logic?
Can we check how other tasks are doing it or if something is available in lib code?

File: Tasks/AzureTestPlanV0/manualTests.ts [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
+
+    simpleTimer.start();
     try{
         manualTestRun = await createManualTestRun(testPlanInfo);
     }
     catch (err){
         tl.debug(`Unable to create Manual Test Run. Err:( ${err} )`);
         return 1;
     }
+    simpleTimer.stop(ciData);
```
[the remark this answers is not in the document]
Comment (adityashahms) [reply Tasks/AzureTestPlanV0/manualTests.ts:21]: Updated

File: Tasks/AzureTestPlanV0/SimpleTimer.ts (hunk 2 above)
Comment (adityashahms) [reply Tasks/AzureTestPlanV0/SimpleTimer.ts:1]: Other are doing start and end at each function block. Hence, pulled out the same and implemented the simple timer class to help with reusability

File: Tasks/AzureTestPlanV0/constants.ts (hunk 1 above)
Comment (adityashahms) [reply Tasks/AzureTestPlanV0/constants.ts:14]: Updated

**Review by triptijain2112 — APPROVED** [review]

**Pull request merged**